TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/Timeline/Events/EventWithUser.cs

```diff
@@ -0,0 +1,23 @@
+namespace TeachingRecordSystem.SupportUi.Pages.Persons.PersonDetail.Timeline.Events;
+
+/// <summary>
+/// Flattened out record to allow Event, TRS User and DQT User to be returned in a single SQL query
+/// </summary>
+public record EventWithUser
```

Comment:
Could this be `private` within `ChangeLog.cshtml.cs`?

Reply:
Done